examples/test/post-processing/components/envmap.js

```diff
@@ -0,0 +1,14 @@
+/* global AFRAME THREE */
+
+AFRAME.registerComponent('envmap', {
+
+  update: function () {
+    this.el.envEl = document.querySelector('a-videosphere');
+  },
+
+  tick: function () {
+    var et = this.el.envEl.object3DMap.mesh.material.map;
```

Comment:
yeah I'll change it to environmentMap. Is there an event emmited somewhere that would indicate that the videosphere is ready with mesh and material? If we could catch something like that we could get rid of the tick method here, the mapping only needs to be set once